elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -1359,7 +1370,7 @@ private Resource toResource(final Supplier<Map<String, Relationship>> relationsh
      *
      * @return Relationship mapping
      */
-    protected Map<String, Relationship> getRelationships() {
+    public Map<String, Relationship> getRelationships() {
```

Comment:
I'm not sure about how I feel about making more methods here public just for tests.  This is one of the classes that has so many touch points - we really want to keep the contract locked down.

Reply:
This was by mistake. fixed